OpenRA.Mods.Common/Activities/Air/Land.cs

```diff
@@ -19,20 +19,24 @@ public class Land : Activity
 	{
 		readonly Target target;
 		readonly Aircraft aircraft;
-		readonly bool requireSpace;
-		readonly Actor ignoreActor;
+		readonly WVec offset;
 
 		bool landingInitiated;
 		bool soundPlayed;
 
-		public Land(Actor self, Target t, bool requireSpace, Actor ignoreActor = null)
+		public Land(Actor self, Target t, WVec offset)
 		{
 			target = t;
 			aircraft = self.Trait<Aircraft>();
-			this.requireSpace = requireSpace;
-			this.ignoreActor = ignoreActor;
+			this.offset = offset;
 		}
 
+		public Land(Actor self, Target t)
+			: this(self, t, WVec.Zero) { }
+
+		public Land(Actor self)
+			: this(self, Target.FromPos(self.Trait<Aircraft>().GroundPosition), WVec.Zero) { }
```

Comment:
`GroundPosition` is only used from here, and doesn't actually depend on `Aircraft`. You can define a `static WPos GroundPosition(Actor self)` method here and then do `Target.FromPos(GroundPosition(self))` instead.

Reply:
Went with the latter.